docs/chap-networking.md

```diff
@@ -214,6 +214,158 @@ The Polkadot Host must actively communicate with the network in order to partici
+The response to the previous query includes an enum with one of the following values:
+<!-- Option<HashSet<AuthorityDiscoveryId> -->
+
+| Value                  | Description                                            |
+|------------------------|--------------------------------------------------------|
+|  None                  | A type representing `no value`                           | 
+|  HashSet[authorityId]  | An unordered collection of unique `authorityId` elements | 
+
+
+**with**
+- `authorityId` the 256-bit identifier representing the public key of the requested `PeerId`. 
+
+:::
+
+
+<!-- AUTHORITY DISCOVER interacting with KADEMLIA -->
+#### -sec-num- Publishing local addresses {#sect-auth-discovery-worker-publishing}
+<!-- worker component: network.put_value( hash(authorityId) , SignedAuthorityRecord);  -->
+
+The authority discovery mechanism periodically sends `put_value` messages to the DHT in order to publish a `SignedAuthorityRecord` of the addresses of authorities it knows from its current authority sets. The `put_value` message to be sent to Kademlia DHT is created as follows.
+<!-- Retrieve authority identifiers of the current and next authority set.. -->
+
+###### Definition -def-num- Signed Authority Record {#defn-msg-signed-authority-record}
+:::definition
+
+The `SignedAuthorityRecord` is a Protobuf serialized structure representing the authority records and signature to send over the wire.
+It is defined in the following format:
+
+| Type               | Id  | Description                                           |
+|--------------------|-----|-------------------------------------------------------|
+| *AuthorityRecord*  | 1   | Serialized authority record                           |
+| `bytes`            | 2   | An Schnorrkel/Ristretto x25519 ("sr25519") signature  |
+| *PeerSignature*    | 3   | Serialized peer signature                             |
+
+**where**  
+
+**AuthorityRecord** is a serialized Protobuf structure that lists the addresses of authority nodes that are currently part of the authority set. 
+
+| Type             | Id | Description                                                           |
+|------------------|----|-----------------------------------------------------------------------|
+| `repeated bytes` | 1  | Possibly multiple `MultiAddr`ess through which a node can be connected | 
+
+**PeerSignature** is a Protobuf serialized structure indicating the signature and public key used to sign and verify the `AuthorityRecord`.
+This is the protobuf structure used to exchange the signature with other nodes.
+
+| Type    | Id  | Description                                       |
+|---------|-----|---------------------------------------------------|
+| `bytes` | 1   | An sr25519 signature                              | 
+| `bytes` | 2   | A sr25519 public key used to verify the signature | 
+
+:::
+
+
+###### Definition -def-num- Publish authority addresses message {#defn-msg-auth-discovery-publish}
```

Comment:
I'm a bit confused by this section. Isn't it the same thing as in the previous section?